haas/migrations/versions/f4a1f5d7ec43_network_acl.py

```diff
@@ -0,0 +1,49 @@
+"""network ACL
+
+Revision ID: f4a1f5d7ec43
+Revises: 6a8c19565060
+Create Date: 2016-04-26 13:34:54.910541
+
+"""
+
+# revision identifiers, used by Alembic.
+revision = 'f4a1f5d7ec43'
+down_revision = None
```

Comment:
Not certain that this is your problem, but down_revision should match the "Revises" line in the docstring. The migrations doc says to change it to None only for new extensions, not for haas core. Perhaps that difference should be called out explicitly.